pygeos/test/test_linear.py

```diff
@@ -64,3 +64,11 @@ def test_line_merge_geom_array():
     actual = pygeos.line_merge([line_string, multi_line_string])
     assert pygeos.equals(actual[0], line_string)
     assert pygeos.equals(actual[1], multi_line_string)
+
+def test_shared_paths_linestring():
+    g1 = pygeos.linestrings([(0, 0), (1, 0), (1, 1)])
+    g2 = pygeos.linestrings([(0, 0), (1, 0)])
+    actual1 = pygeos.shared_paths(g1, g2)
+    assert pygeos.equals(pygeos.get_geometry(actual1, 0), g2)
+    assert pygeos.shared_paths(None, None) == None
```

Comment:
```suggestion
    assert pygeos.shared_paths(None, None) is None
```

(in practice doesn't matter much, but for checking to None, using `is` is seen as better practice)